test/python/apps/exchange.py

```diff
@@ -191,15 +195,20 @@ def check_address(self, payout_signer: SigningAuthority, refund_signer: Optional
             with self._exchange_async(Command.CHECK_PAYOUT_ADDRESS, payload=payload) as response:
                 yield response
 
+
```

Comment:
Linters will yield at you for these extras empty lines between methods of a class.